subsys/bluetooth/mesh/transport.c

```diff
@@ -467,7 +467,7 @@ static void seg_tx_send_unacked(struct seg_tx *tx)
 
 end:
 	if (IS_ENABLED(CONFIG_BT_MESH_LOW_POWER) &&
-	    bt_mesh_lpn_established()) {
+	    bt_mesh_lpn_established() && !bt_mesh_has_addr(ctx.addr)) {
```

Comment:
The polling is needed because the transport layer requires ack. If LPN will wait for the next poll timeout, the segmented message transmission will most likely be aborted.